app/src/main/java/org/oppia/app/utility/ClickableAreasImage.kt

```diff
@@ -0,0 +1,112 @@
+package org.oppia.app.utility
+
+import android.graphics.RectF
+import android.view.View
+import android.widget.FrameLayout
+import android.widget.ImageView
+import androidx.core.view.forEachIndexed
+import androidx.core.view.isVisible
+import com.github.chrisbanes.photoview.OnPhotoTapListener
+import com.github.chrisbanes.photoview.PhotoViewAttacher
+import org.oppia.app.R
+import org.oppia.app.player.state.ImageRegionSelectionInteractionView
+import kotlin.math.roundToInt
+
+class ClickableAreasImage(
```

Comment:
Please add a KDoc for this class & all other public API members in this class & other classes.

Reply:
Done